indico_toolkit/indico_wrapper/pred_manipulation.py

```diff
@@ -0,0 +1,115 @@
+from typing import List
+
+
+class ManipulatePreds:
+    """
+    Class to help expand prediction text and indexing via OCR data.
+    """
+
+    def __init__(self, ocr_tokens: List[dict], preds: List[dict]):
+        self.ocr_tokens = ocr_tokens
+        self.predictions = preds
+
+    def expand_predictions(self, pred_start: int, pred_end: int) -> dict:
```

Comment:
I'm not sure this is how I'd imagine using this function-- wouldn't you have a specific prediction in mind where you want this to run? So the function could be static and take list of ocr_tokens and pred and then expands then returns the pred (expanded if that's relevant-- updating the start/end indexes as well as the text value)? Let me know what your rationale was for pred start/end instead I might be missing something

Reply:
"alks Sco"-- > ["talks", "Scott"] - > "talks Scott"

Reply:
def expand_pred(some_pred: dict, tokens: List[dict]) -> expanded_pred (dict)

Reply:
-> dict, bool (bool = True if updated)